abi/erc20.go

```diff
@@ -1,5 +1,56 @@
-package abis
+package contracts
 
+import (
+	"errors"
+	"math/big"
+	"strconv"
+)
+
+type Erc20Params struct {
+	Symbol      string
+	TokenName   string
+	Decimals    int
+	TotalSupply *big.Int
+}
+
+func ParseERC20Params(args []string) (*Erc20Params, error) {
```

Comment:
Parsing the command line args should be done in main, and this func should be normal idiomatic, type-safe go. For example, I plan to call this from chainload.

Reply:
Er, this only parses, so maybe it just goes away, or is replaced by something that wraps executing the template.